packages/styling/src/styles/DefaultFontStyles.ts

```diff
@@ -203,10 +200,10 @@ function _registerDefaultFontFaces(): void {
  * Reads the fontBaseUrl from window.FabricConfig.fontBaseUrl or falls back to a default.
  */
 function _getFontBaseUrl(): string {
-  let win = getWindow();
+  let win: {} = typeof window !== 'undefined' ? window : undefined;
 
   // tslint:disable-next-line:no-string-literal no-any
-  let fabricConfig: IFabricConfig = win ? (win as any)['FabricConfig'] : undefined;
+  let fabricConfig: IFabricConfig = win ? win['FabricConfig'] : undefined;
```

Comment:
Because FabricConfig can be burned on the page by the server easily, whereas GlobalSettings doesn't read from the page on startup. We *could* add that though. Let's talk about this, and the other suggestions tomorrow and come up with the right plan.